README.md

```diff
@@ -192,6 +192,7 @@ Elasticsearch-Evolution can be configured to your needs:
 -   **placeholderSuffix** (default=}): Suffix of placeholders in migration scripts.
 -   **historyIndex** (default=es_evolution): Name of the history index that will be used by Elasticsearch-Evolution. In this index Elasticsearch-Evolution will persist his internal state and tracks which migration script has already been executed.
 -   **historyMaxQuerySize** (default=1000): The maximum query size while validating already executed scripts. This query size have to be higher than the total count of your migration scripts.
+-   **rejectModifiedScripts** (default=true): Whether to fail when a previously applied migration script has been modified after it was applied.
```

Comment:
In flyway this property is called `validateOnMigrate`, see https://flywaydb.org/documentation/configuration/parameters/validateOnMigrate and https://flywaydb.org/documentation/command/validate
I would prefer to name the property as in flyway. This makes it easier to work with elasticsearch-evolution when you are familar with flyway.